### PR TITLE
Release authentification

### PR DESCRIPTION
**Pull request by Jonathan-dehaay**
Add condition to launch authentification on debug or release mode.

### CODE REVIEW

File: Views/PersonCreationView.swift [hunk 1]
```diff
@@ -38,7 +38,7 @@ class PersonCreationView: UIView, UITextFieldDelegate {
     let firstNameErrorLabel = UILabel(frame: CGRect.zero)
     firstNameErrorLabel.text = "person_first_name_is_empty".localized
     firstNameErrorLabel.font = UIFont.systemFont(ofSize: 13)
-    firstNameErrorLabel.textColor = AppColor.AppleColors.Red
+    firstNameErrorLabel.textColor = AppColor.TextColors.Red
```
Comment (Aquaj) [inline Views/PersonCreationView.swift]: Don't we want to use `AppleColor`s ?

File: Views/PersonCreationView.swift [hunk 2]
```diff
@@ -62,7 +62,7 @@ class PersonCreationView: UIView, UITextFieldDelegate {
     let lastNameErrorLabel = UILabel(frame: CGRect.zero)
     lastNameErrorLabel.text = "person_last_name_is_empty".localized
     lastNameErrorLabel.font = UIFont.systemFont(ofSize: 13)
-    lastNameErrorLabel.textColor = AppColor.AppleColors.Red
+    lastNameErrorLabel.textColor = AppColor.TextColors.Red
```
Comment (Aquaj) [inline Views/PersonCreationView.swift]: Don't we want to use `AppleColor`s ?

File: Views/PersonCreationView.swift (hunk 1 above)
Comment (Jonathan-dehaay) [reply Views/PersonCreationView.swift]: Of course, it's a merge error.

File: Views/PersonCreationView.swift (hunk 2 above)
Comment (Jonathan-dehaay) [reply Views/PersonCreationView.swift]: Of course, it's a merge error.

**Pull request merged**